packages/interfaces/src/message.ts

```diff
@@ -60,3 +60,9 @@ export interface IDecoder<T extends IDecodedMessage> {
   fromWireToProtoObj: (bytes: Uint8Array) => Promise<IProtoMessage | undefined>;
   fromProtoObj: (proto: IProtoMessage) => Promise<T | undefined>;
 }
+
+// Non-empty array of decoders
```

Comment:
I think it is not possible to guarantee

Reply:
Good point, indeed it's likely to help in some scenarios (using TS with an array obviously empty) but not that much. Will revert the change as it clutters the code for little benefit.